docker-compose.yml

```diff
@@ -0,0 +1,136 @@
+version: '3.8'
+
+services:
+  database:
+    image: "${DOCKER_REPO}aspen-devdb:latest"
```

Comment:
where does this image come from?  ECR?  how does it get updated?

Reply:
Yep, ECR. At the moment it's updated manually, but we'll need to work out a plan for how/when to refresh it.

Reply:
Do you have a list of future work you're tracking?

Reply:
how often do you think this would need to be updated?